src/modal_server.py

```diff
@@ -26,6 +26,8 @@
 # Inactivity timeout in minutes
 INACTIVITY_TIMEOUT = 30
 
+data_handler = DataHandler('root') # HERE PLEASE MODIFY THR root TO RESPECTIVE VALUE
```

Comment:
Use modal volumes for this: https://modal.com/docs/guide/volumes, for the modal setup.

Reply:
used modal volumes in new update